src/reference/asciidoc/gateway.adoc

```diff
@@ -709,7 +709,7 @@ String out = result.get(10, TimeUnit.SECONDS);
 ----
 ====
 
-===== Reactor `Mono`
+===== Reactor Mono
```

Comment:
This was @Buzzardo 's change.
His point is like express code even in the titles.

Reply:
Ah - I was looking at the release version and it appeared it didn't show up.

However, there is something seriously wrong with the ToC - lots of missing sections - no Section V (until appendix and lots of missing sections)

https://docs.spring.io/spring-integration/docs/5.1.0.BUILD-SNAPSHOT/reference/html/

Reply:
Yeah... I've seen that too. Time to review it one more time?

/CC @Buzzardo